app/Http/Controllers/UserListController.php

```diff
@@ -8,11 +8,11 @@
 
 class UserListController extends Controller
 {
-    public function create(){
+    public function create() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->